fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -436,17 +444,26 @@ public int getCount() {
          * @return the cumulative time spent on the associated event
          */
         public long getTimeNanos() {
-            return timeNanos.get();
+            return getCumulativeValue();
```

Comment:
We should probably note that this method only returns the time for time-based events. The backwards compatibility argument for leaving this in at all rather than splitting event into `TimeEvent` and `SizeEvent` makes sense to me, but we probably want to spell that out in documentation for clarity's sake.